### ISSUE TITLE
mouse input seems...wonky? the mouse does not move the R-stick properly especially on the y axis, causing the input to not translate correctly, sometimes even moving down for up and vice versa

### ISSUE THREAD

**Issue by hMan1137** [luttje/Key2Joy#83]
### Discussed in https://github.com/luttje/Key2Joy/discussions/55

<div type='discussions-op-text'>

<sup>Originally posted by **luttje** November  1, 2023</sup>
## 👋 Welcome!
We’re using Discussions as a place to connect with other members of our community. GitHub discussions are like a small forum where you make a discussion (a.k.a. thread/topic) to which others can reply.

We hope that you:
  * [Ask questions you’re wondering about.](https://github.com/luttje/Key2Joy/discussions/categories/q-a)
  * Engage with other community members.
  * [Share ideas for features](https://github.com/luttje/Key2Joy/discussions/categories/ideas)
  * [Share something cool you made](https://github.com/luttje/Key2Joy/discussions/categories/show-and-tell)
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## 🐞 Found a bug?
Please [create an issue under 'Issues'](https://github.com/luttje/Key2Joy/issues) so we can track progress on fixing them. 

Feel free to use Discussions to explore with us whether you've found a bug, or are overlooking something.

## ❤ Thanks for your input!
We appreciate your bug reports, contributions, help or other positive/constructive interactions and feedback. With your help we can make this free and open-source software useful to all who need it.</div>

**Comment by luttje** [comment]
You're absolutely right, there's other issues reporting this too. Sorry it's not better atm.

I've not yet found the motivation to work on this project again however, so a fix is not in sight yet.
